app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
+  def create
+    puts comment_params
+    @comment = Comment.new comment_params
+    puts 'VALID?: ', @comment.valid?
+    @comment.save if @comment.valid?
+    redirect_to post_path(@comment.post)
+  end
+
+  private
+
+  def comment_params
+    params.require(:comment).permit(:text).merge(user: current_user, post_id: params.require(:post_id))
+  end
```

Comment:
- [ ] OPTIONAL: You don't have to add user id comment params. it can be added directly in create action, please change it to: 

``params.require(:comment).permit(:text)``

 In controller to add a user id in record you can do as follows:

`@comment = @current_user.comments.new(comment_params)`